app/src/main/res/layout/item_update_card.xml

```diff
@@ -82,9 +82,16 @@
           android:layout_marginTop="@dimen/grid_3"
           android:ellipsize="end"
           android:importantForAccessibility="no"
-          android:maxLines="4"
-          tools:text="This is some placeholder copy that is only visible when previewing the layout." />
+          tools:text="This is some placeholder copy that is only visible when previewing the layout."
+          android:maxLines="6"/>
       </LinearLayout>
+
+      <ImageView
+          android:layout_width="match_parent"
+          android:layout_height="@dimen/grid_3"
+          android:layout_gravity="bottom"
+          android:background="@drawable/white_gradient"
+          android:contentDescription="@null"/>
```

Comment:
Setting the content description to `null` works but I like to use `android:importantForAccessibility="no"` 
https://developer.android.com/reference/android/view/View#attr_android:importantForAccessibility

Reply:
Done!